Console/Command/NostoAccountConnectCommand.php

```diff
@@ -0,0 +1,242 @@
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Products Token'
+            )->addOption(
+                Token::API_SETTINGS . self::TOKEN_SUFFIX,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Settings token'
+            )->addOption(
+                Token::API_EXCHANGE_RATES . self::TOKEN_SUFFIX,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'API exchange rates token'
+            )->addOption(
+                Token::API_EMAIL . self::TOKEN_SUFFIX,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Email token'
+            )->addOption(
+                self::SCOPE_CODE,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Store view code'
+            );
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->isInteractive = !$input->getOption('no-interaction');
+        $io = new SymfonyStyle($input, $output);
+        $accountId = $input->getOption(self::NOSTO_ACCOUNT_ID) ?:
+            $io->ask('Enter Nosto Account Id: ');
+        $scopeCode = $input->getOption(self::SCOPE_CODE) ?:
+            $io->ask('Enter Store Scope Code');
+
+        $tokens = $this->getTokensFromInput($input, $io);
+        if ($this->updateNostoTokens($tokens, $accountId, $io, $scopeCode)) {
+            $io->success('Tokens Sucessfully Configured');
+        } else {
+            $io->error('Could not complete operation');
+        }
+    }
+
+    /**
+     * Set or override tokens for the given account id.
+     * If a local account is not found, will create a new one.
+     *
+     * @param array $tokens
+     * @param $accountId
+     * @return bool
+     */
+    private function updateNostoTokens(array $tokens, $accountId, SymfonyStyle $io, $scopeCode)
+    {
+        $store = $this->getStoreByCode($scopeCode);
+        if(!$store){
+            $io->error('Store not found. Check your input.');
+            return false;
+        }
+        $storeAccountId = $store->getConfig(NostoAccountHelper::XML_PATH_ACCOUNT);
+        $account = $this->accountHelper->findAccount($store);
+        if ($account && $storeAccountId === $accountId) {
+           // If the script is non-interactive, do not ask for confirmation
+           $confirmOverride = $this->isInteractive ?
+                $confirmOverride = $io->confirm(
+                    'Local Nosto account found for this store view. Override tokens?',
+                    false
+                ):
+               true;
+            if ($confirmOverride) {
+                $account->setTokens($tokens);
+                return $this->accountHelper->saveAccount($account, $store);
+            }
+        } else {
+            $io->note('Local account not found, creating a new one...');
```

Comment:
This might be a bit misleading note. We are not really creating new account are we?